apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page/create-entity-page/entity-type-selector.tsx

```diff
@@ -36,7 +37,10 @@ export const EntityTypeSelector: FunctionComponent<{
       options={entityTypes ?? []}
       optionToRenderData={({ schema: { $id, title, description } }) => ({
         uniqueId: $id,
-        Icon: isLinkTypeLookup?.[$id] ? LinkTypeIcon : EntityTypeIcon,
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- @todo why this false positive?
+        Icon: isSpecialEntityTypeLookup?.[$id]?.link
+          ? LinkTypeIcon
+          : EntityTypeIcon,
```

Comment:
Have you tried again? This seems to work in my editor locally
```suggestion
        Icon: isSpecialEntityTypeLookup?.[$id]?.link
          ? LinkTypeIcon
          : EntityTypeIcon,
```

Reply:
I tried removing this and it caused the [failure in CI](https://github.com/hashintel/hash/actions/runs/6106723602/job/16572323236?pr=3104)

Reply:
Strange, let's leave it then